src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -74,11 +75,33 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.SharedFramework.Sdk" />
 
   <Target Name="ResolveReadyToRunCompilers" DependsOnTargets="ResolveRuntimeFilesFromLocalBuild">
+    <PropertyGroup> <!-- Can be removed once runtime repo switches over to 6.0 drop of the SDK repo -->
+      <JitTargetArchComponent>$(TargetArchitecture)</JitTargetArchComponent>
+      <JitTargetOSComponent>unix</JitTargetOSComponent>
+      <JitTargetOSComponent Condition="'$(TargetOS)' == 'windows'">win</JitTargetOSComponent>
+      <JitBuildArchComponent>$(BuildArchitecture)</JitBuildArchComponent>
+      <JitArchSuffix>$(JitTargetArchComponent)_$(JitBuildArchComponent)</JitArchSuffix>
+      <JitFileName>$(LibPrefix)clrjit_$(JitTargetOSComponent)_$(JitArchSuffix)</JitFileName>
```

Comment:
I would avoid introducing single-use properties:
```suggestion
      <JitTargetOSComponent>unix</JitTargetOSComponent>
      <JitTargetOSComponent Condition="'$(TargetOS)' == 'windows'">win</JitTargetOSComponent>
      <JitFileName>$(LibPrefix)clrjit_$(JitTargetOSComponent)_$(TargetArchitecture)_$(BuildArchitecture)</JitFileName>
```

Reply:
Sounds good, will fix in a bit, I just don't want to restart the build now as I want to see the effect of the Unix change on the build (whether at least something passes now).

Reply:
Fixed in 3rd commit.

Reply:
(Along with a few related cleanups.)